mypy/stubtest.py

```diff
@@ -568,6 +568,13 @@ def verify_typeinfo(
             # Catch all exceptions in case the runtime raises an unexpected exception
             # from __getattr__ or similar.
             continue
+
+        # If it came from the metaclass, consider the runtime_attr to be MISSING
+        # for a more accurate message
+        if runtime_attr is not MISSING and type(runtime) != runtime:
```

Comment:
Should `!=` be `is not`?

And can you add a test that changes behavior with this patch?

Reply:
I think you're right. I'll have a test ready in a little bit.